envergo/static/js/libs/haie_result_actions_banner.js

```diff
@@ -0,0 +1,68 @@
+// a script to add actions on the moulinette result banner
+(function (exports) {
+    'use strict';
+
+    var demarcheForm = document.getElementById("demarche-simplifiee-form");
+    if (demarcheForm) {
+
+        demarcheForm.addEventListener("submit", function (event) {
+            event.preventDefault();
+
+            // Replace form button by a loader
+            const submitButton = document.getElementById("demarche-simplifiee-btn");
+            let originalButton = null;
+            const textNode = document.createTextNode('Redirection vers Démarches simplifiées...');
```

Comment:
Niveau de pinaillage maximal :muscle: 

... -> …

```suggestion
            const textNode = document.createTextNode('Redirection vers Démarches simplifiées…');
```